alodataset/coco_base_dataset.py

```diff
@@ -231,7 +238,14 @@ def getitem(self, idx):
             return BaseDataset.__getitem__(self, idx)
 
         image_id = self.items[idx]
+        if "test" in self.img_folder:
+            #get the filename from image_id without relying on annotation file
+            frame = Frame(os.path.join(self.img_folder, f"{str(image_id).zfill(12)}.jpg"))
```

Comment:
return directly, no need to assign
`return Frame(os.path.join(self.img_folder, f"{str(image_id).zfill(12)}.jpg"))`

Reply:
I actually commited this file by mistake, it was already merged in #247